relationships/synthesis/EXT-SERVICE-to-INFRA-ELASTICACHEMEMCACHEDCLUSTER.yml

```diff
@@ -0,0 +1,56 @@
+relationships:
+  - name: extServiceCallsInfraElasticacheMemcachedCluster
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
```

Comment:
Looks like we have data matching this condition: https://staging.onenr.io/0qwyDK3ozQn

Reply:
Attributes are also present in the `Span` events: https://staging.onenr.io/0ERPpka1YRW
Is there any reason to use `Metric` over `Span`? Could we use both?

Reply:
Oh, my bad!I did not check Span event for the matching data hence only used Metric event here.  
I think we can use both. But, could you please let me know why we would need both the events here if one of the events data contains the required attributes? Will that make it more optimised?

Reply:
I'm not sure if there are scenarios where just `Metric` or `Span` is reported. If that the case, adding both event types covers all the cases.

Could you check with the agent team if it's a valid case?

Reply:
It is a valid case. I have added Span along with Metrics.

Reply:
As per some internal discussions, keeping the event type limited to "Metric" here.
From the COGS perspective, it is better to use only one, instead of specifying both. From the agent side there is no real preference. But, Metrics could be little better because they should always be reported, while Spans can be disabled if DT is turned off.